docs/source/aws/examples/asgnotused.rst

```diff
@@ -0,0 +1,59 @@
+AMI - ASG Garbage Collector
+====================================
+ASG garbage collector which mean that:
+
+Check if an ASG has minSize = 0 and DesiredCapacity = 0
+Mark the ASG as ops to alert.
+If value won't change cloudCustodian will send an alert with ASGs.
+
+.. code-block:: yaml
+
+
+  - name: asg-mark-as-unused
+    resource: asg
+    comments: |
+      Mark any unused ASG checking it every day.
+    filters:
+      - type: value
+        key: MinSize
+        value: 1
+        op: less-than
```

Comment:
this could probably be an eq. Can the number ever be negative? I don't think so.